openfisca_france/parameters/prestations_sociales/prestations_etat_de_sante/invalidite/pensions_invalidite/revalorisations_invalidite.yaml

```
@@ -0,0 +1,164 @@
+description: Coefficient de revalorisation des pensions d'invalidité du régime général
+values:
+  1998-01-01:
+    value: 1.011
+  1999-01-01:
+    value: 1.012
+  2000-01-01:
+    value: 1.005
+  2001-01-01:
+    value: 1.022
+  2002-01-01:
+    value: 1.022
+  2003-01-01:
+    value: 1.015
+  2004-01-01:
+    value: 1.017
+  2005-01-01:
+    value: 1.02
+  2006-01-01:
+    value: 1.018
+  2007-01-01:
+    value: 1.018
+  2008-01-01:
+    value: 1.011
+  2008-09-01:
+    value: 1.008
+  2009-04-01:
+    value: 1.01
+  2010-04-01:
+    value: 1.009
+  2011-04-01:
+    value: 1.021
+  2012-04-01:
+    value: 1.021
+  2013-04-01:
+    value: 1.013
+  2014-04-01:
+    value: 1.006
+  2015-04-01:
+    value: 1
+  2016-04-01:
+    value: 1.001
+  2017-04-01:
+    value: 1.003
+  2018-04-01:
+    value: 1.01
+  2019-04-01:
+    value: 1.003
+  2020-04-01:
+    value: 1.009
+  2021-04-01:
+    value: 1.001
+  2022-04-01:
+    value: 1.018
+  2022-07-01:
+    value: 1.04
+  2023-04-01:
+    value: 1.015385
+  2024-04-01:
+    value: 1.046
+metadata:
+  short_label: Indexation des pensions d'invalidité
+  label_en: Disability pension indexation
+  short_label_en: Disability pension indexation
+  ipp_csv_id: revalorg
+  reference:
+    1998-01-01:
+      title: Arrêté du 29/12/1997
+      href: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000000387479/
+    1999-01-01:
+      title: Arrêté du 29/12/1998
+      href: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000000757782/
+    2000-01-01:
+      title: Arrêté du 29/12/1999
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000762866
+    2001-01-01:
+      title: Arrêté du 26/12/2000
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000588355
+    2002-01-01:
+      title: Arrêté du 25/01/2002
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000399330
+    2003-01-01:
+      title: Arrêté du 31/12/2002
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000776880
+    2004-01-01:
+      title: Arrêté du 23/12/2003
+      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000776880
```

Lien de l'année précédente
```suggestion
      href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000000611637
```